src/govuk/components/accordion/_index.scss

```diff
@@ -263,6 +263,15 @@
       &:active {
         background: $govuk-link-active-colour;
       }
+
```

Comment:
We discussed that this commit https://github.com/alphagov/govuk-frontend/pull/2257/commits/4d44935136f9470d851fb6f7e6e37998f3980176# 
might need 
- a bit more detail on the original problem that we're fixing
- the desired outcome
- to be split in two as there are two slightly different changes here

It would also be good to dig out the original bug report for NVDA - @chris-gds let me know if I can help with that.

Reply:
Agreed, updating commit with details splitting out commit thank contains pause 

-----

Additional Details for context 
Original issue flagged:

Not sure if this is a quirk of NVDA, but if you navigate through the accordion (arrow keys), it reads out the heading (together with ‘Show’) and ‘this section’ as 2 separate items, which could be confusing:

_clickable  heading    level 2  button  collapsed    Know your audience,Show
heading    level 2  button  collapsed    this section_

---- 

Clinic Notes:
"Notably, we didn’t take issue with when NVDA split the button into separate blocks. It’ll be good regardless to get some detailed rationale as to why this could be acceptable and how theoretically impactful the lack of pausing is.

The rationale was NVDA has been seen to treat headings / buttons in a similar fashion elsewhere on the web. There would be a degree of expectation that the user might be used to this."

---

Action -  agreed to fix issue.

Reply:
- Added more detail to commits about the history of being spotted, feedback, investigation and fix also,
- Added inline comments to further outline issue.
- PR also updated with before and after videos for visual reference of the issue
- Pause split out of original commit for clarity